contrib/lang/ruby/packages.el

```diff
@@ -41,15 +47,82 @@ which require an initialization must be listed explicitly in the list.")
 (defun ruby/init-flycheck ()
   (add-hook 'enh-ruby-mode-hook 'flycheck-mode))
 
+(defun ruby/init-projectile-rails ()
+  (use-package projectile-rails
+    :defer t
+    :init (progn
+            (add-hook 'projectile-mode-hook 'projectile-rails-on))
+    :config (progn
+              (spacemacs|diminish projectile-rails-mode " ⇋" " R")
+              ;; GOTO commands
+              (evil-leader/set-key "mrf" 'projectile-rails-goto-file-at-point)
```

Comment:
sort the key bindings and apply the conventions inside the prefix command `mr`. It will lead to 4 characters key bindings but with `,` it is reduced to 3 so we are okay, moreover this is the approach taken for Django.